pkg/controller/config/config_controller.go

```diff
@@ -103,6 +103,10 @@ func readAddons(mgr manager.Manager) (mf.Manifest, error) {
 	if err != nil {
 		return mf.Manifest{}, err
 	}
+	if runtime1.GOARCH == "ppc64le" || runtime1.GOARCH == "s390x" {
+                log.Info("skip installation of tektoncd/catalog tasks if the platform is not x86_64")
```

Comment:
a statement would be better, instead of a conditional.

how about
```
log.Info("skipping installation of tektoncd/catalog tasks as the platform is not x86_64")
```